src/components/TabsPage.tsx

```diff
@@ -0,0 +1,27 @@
+import { useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+import { Tabs } from './Tabs';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+
+  const selectedTab = tabs.find(tab => tabId === tab.id);
```

Comment:
This logic is repeated twice in the Tabs component and in TabsPage.
Use it once